.github/release_build.py

```
@@ -34,7 +34,7 @@ def get_blob(branch):
 
     subprocess.run(['git', 'push', 'origin', 'master'], check=True)
 
-    subprocess.run('cd build; sha256sum update_all.pyz > update_all.pyz.sha256', shell=True, check=True)
+    subprocess.run(['sha256sum', 'build/update_all.pyz'], stdout=open('build/update_all.pyz.sha256', 'w'), check=True)
```

Ruff (0.16.0)

[error] Starting a process with a partial executable path

(S607)